mytimer/params.py

```diff
@@ -4,7 +4,6 @@
 MY_TIMER_VERSION = "1.2"
 ADDITIONAL_INFO = "Additional information: Press `Ctrl+C` to exit."
 INPUT_ERROR_MESSAGE = "[Error] Wrong input"
-SOUND_ERROR_MESSAGE = "[Error] Unable to play sound"
```

Comment:
Why did you delete this line?
This program is a CLI and should not raise an error in this particular scenario.